frontend/src/templates/Field/Section/SectionField.tsx

```diff
@@ -16,35 +16,17 @@ export interface SectionFieldProps extends SectionFieldContainerProps {
 
 // Used by SectionFieldContainer
 export const SectionField = forwardRef<SectionFieldProps, 'div'>(
-  ({ schema, colorTheme = FormColorTheme.Blue, handleSectionEnter }, ref) => {
-    const sectionColor = useMemo(() => {
-      switch (colorTheme) {
-        case FormColorTheme.Orange:
-        case FormColorTheme.Red:
-          return `theme-${colorTheme}.600` as const
-        default:
-          return `theme-${colorTheme}.500` as const
-      }
-    }, [])
-
+  ({ handleSectionEnter, ...rest }, ref) => {
     return (
       <Box
         _notFirst={{
           mt: '3.75rem',
         }}
       >
-        {/* id given so app can scrolled to this section */}
-        <Box id={schema._id} ref={ref}>
-          <Text textStyle="h2" color={sectionColor}>
-            {schema.title}
-          </Text>
-          <Text textStyle="body-1" color="secondary.700" mt="1rem">
-            {schema.description}
-          </Text>
-        </Box>
+        <BaseSectionField {...rest} ref={ref} />
         <Waypoint
           topOffset="80px"
-          bottomOffset="70%"
+          bottomOffset="0%"
```

Comment:
/superNit: size 0 doesn't need unit or percent?
```suggestion
          bottomOffset="0"
```

Reply:
lazy hehe